### DISCUSSION TITLE
Colors in PCM?

### DISCUSSION THREAD

**Discussion by Erotemic** [opcm/pcm#804] [Q&A]
I'm on Ubuntu 22.04 with an i9-11900K, and I'm interested in using pcm to determine if data transfer is a bottleneck in my applications. Some googling pointed me to this application as a way to get some insight. However, walls of text are hard to read, and I would like to have a nice looking output in the terminal.

In the README, there is an image where the PCM output in the terminal has ANSI coloring, which makes it much easier to read. However, when I run:

```
sudo apt install pcm
sudo pcm
```

All of the text I get on screen has the same color. I searched the `--help` and environment variable documentation as well as the FAQ to see if there was something I had to do to enable colors, but I didn't see anything.

Is there a way that I can enable colored output? Is this possible with the `apt` version of the tool? Do I need to build from source with special compile time operations?

EDIT: I did compile from source, but still didn't see any way to enable colors. 

EDIT: It also looks like I can't use `pcm-pcie` on a rocket lake architecture :disappointed:. I'm getting `Jaketown, Ivytown, Haswell, Broadwell-DE Server CPU is required for this tool! Program aborted`.

**Comment by rdementi** [comment]
yes, you are right. The version from apt is not the latest. Please build from source on github and also use the --color option (--help suggests it). For the rocket lake it is a known limitation: https://github.com/intel/pcm/blob/master/doc/FAQ.md#q12
